README.md

```diff
@@ -1 +1,90 @@
-# atracker
\ No newline at end of file
+# IBM Cloud Atracker - Terraform Module
```

Comment:
Please follow the design guidelines prescribed in (https://github.com/terraform-ibm-modules/getting-started/blob/master/design_guidelines.md) and make the module as Type-1 module

as part of type-1 module, please add root module

Reply:
Root module is the atracker or are you referring to the cos and bucket instances?